internal/devbox/process_compose.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2024 Jetify Inc. and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package devbox
+
+import (
+	"context"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"go.jetpack.io/devbox/internal/devbox/devopt"
+	"go.jetpack.io/devbox/internal/xdg"
+)
+
+const processComposeVersion = "1.5.0"
+
+var processComposeConfigPath string
+
+func InitializeProcessCompose(ctx context.Context, stderr io.Writer) (string, error) {
+	pcDevboxProjectPath, err := ensurePcDevboxProject()
+	if err != nil {
+		return "", err
+	}
+
+	box, err := Open(&devopt.Opts{
+		Dir:    pcDevboxProjectPath,
+		Stderr: stderr,
+	})
+	if err != nil {
+		return "", errors.WithStack(err)
+	}
+
+	if err = box.Add(ctx, []string{"process-compose@" + processComposeVersion}, devopt.AddOpts{}); err != nil {
+		return "", err
+	}
+
+	err = box.Install(ctx)
+	if err != nil {
+		return "", err
+	}
+
+	return utilityLookPath("process-compose")
```

Comment:
Are you sure this is working? I think this may be finding the old process-compose.

Reply:
Yup you're right. I manually deleted the process-compose utility and then re-ran it and it failed. I'll fix it to point to the correct binary and update the PR.